reference-architecture/rhv-ansible/playbooks/openshift-install.yaml

```diff
@@ -60,4 +58,35 @@
         - "role={{ openshift_node_labels.role }}"
     openshift_registry_selector: "role=infra"
     openshift_router_selector: "role=infra"
+    # Load balancer config
```

Comment:
But then it is a SPOF for both API and apps... I think we should discuss if we want a single HAproxy or an HA deployment instead and split API and apps